### ISSUE TITLE
ModuleNotFoundError: No module named '__version__'

### ISSUE THREAD

**Issue by vi** [harelba/q#219]
Can't run pre-built Linux version of `q`:

```
$ wget https://github.com/harelba/q/releases/download/2.0.10/q-x86_64-Linux
$ chmod +x q-x86_64-Linux
$ mv q-x86_64-Linux q
$ ./q
Traceback (most recent call last):
  File "bin/q.py", line 33, in <module>
ModuleNotFoundError: No module named '__version__'
[9649] Failed to execute script q
```

**Comment by harelba** [comment]
True, sorry, 2.0.10 is just a workaround against some build issues. I'll fix that. 

The official current version is 2.0.9 - Please download it from the web site `http://harelba.github.io/q/`, not from releases

**Comment by harelba** [comment]
Removed the 2.0.10 binaries from the releases list, and put a message there until it's fixed and a new version is out.

Please use the official installation page in the [website](http://harelba.github.io/q/)

**Comment by vi** [comment]
Shall 2.0.10 be marked as "Pre-release" on Github? Now it shows green badge "Latest release" on 2.0.10.

**Comment by harelba** [comment]
@vi you're right. Moved 2.0.10 to be a pre-release. thanks.

**Comment by harelba** [comment]
Fixed all the versioning issues, and issues `2.0.16` which fixes this problem - Please download and install it from the website. The issue should not happen anymore.
